polyfills.d.ts

```diff
@@ -2,6 +2,7 @@ interface Window {
   _paq: Array<string[]>
   sevenTag: any
   heatmapEvent: Event
+  PPASHeatmapClickEvent: Event
```

Comment:
I'm not sure whether it's a good idea to include `PPAS` as a prefix. We already have e.g. `_ppms` that became legacy. What do you think about using `PP` instead, as the company's name is not likely to change?